### ISSUE TITLE
Firefox Mobile website has funny button placement

### ISSUE THREAD

**Issue by Fuhrmaaj** [FranceJ/AsgardHacks#5]
The two buttons at the bottom of the page are only a couple pixels off the bottom.

I tested on Chrome's developer tools, chrome, edge, Chrome mobile, and Firefox mobile and it turns out it's just Firefox mobile that has the problem. I think what's happening is that Firefox renders the page, pushes it down to render the browser top bar, then figures there's nothing below and doesn't have a scroll bar. Chrome does something similar, but lets you scroll down if you want.

Completely functional, so it depends on how OCD you are.

**Comment by FranceJ** [comment]
I agree, that totally looks bad. It looks like it can be solved by applying `display: flex` to things like main. The problem right now is that the margin of child elements is allowed to escape the main container and it's messing with the flex math of the body.
